includes/subscriptions/templates/html-subscriptions-plugin-notice.php

```diff
@@ -21,21 +21,23 @@
 						<?php
 							printf(
 								// Translators: placeholders are opening and closing strong HTML tags.
-								esc_html__( 'By deactivating the %1$sWooCommerce Subscriptions%2$s plugin, your store will switch to using the subscriptions functionality %1$sbuilt into WooCommerce Payments%2$s.', 'woocommerce-payments' ),
+								esc_html__( 'By deactivating the %1$sWooCommerce Subscriptions%2$s plugin, your store will switch to using the subscriptions functionality %1$sbuilt into WooCommerce Payments%2$s.%1$s%3$sLearn more.%4$s%2$s', 'woocommerce-payments' ),
```

Comment:
Looks like we're missing a space between the first sentence and "Learn more".

Reply:
Good catch, fixed it in 04f5c1cd882826c024a82135ad1d417a308b8666